phases/ephemeral/witx/typenames.witx

```diff
@@ -552,8 +552,6 @@
 ;; The contents of a $subscription_t when type is `EVENTTYPE_CLOCK`.
 (typename $subscription_clock_t
   (struct
-    ;; The user-defined unique identifier of the clock.
-    (field $identifier $userdata_t)
```

Comment:
This may theoretically be useful for debuggers.

Reply:
What would it add beyond the `clock_id` field?

Reply:
I though I read that it was a string. In that case, a debugger could show that string to the user. I think it would be better to design such debugger integrations separately and for all things it makes sense for at a later time.